test/controllers/application_controller_subclass_test.rb

```diff
@@ -255,6 +255,20 @@ class TestableResourcesControllerTest < ActionController::TestCase
       assert_template :partial => false
     end
   end
+
+  context 'logged in user is deleted' do
+    setup do
+      @user = users(:admin)
```

Comment:
This setup block's rather superfluous, `@user` can be a plain inline variable in the test itself (if needed) instead of an instance variable.